charts/platform-system/templates/flux/istio/_istio.tpl

```diff
@@ -0,0 +1,65 @@
+{{ define "platform-system.istio.repository.name" }}
+{{- printf "%s-%s" .Release.Name .Values.apps.istio.chart.source.repository | replace "+" "_" | trunc 63 | trimSuffix "-" }}
```

Comment:
I don't like truncates on things like this because you end up with a much higher likelihood of a collision.